supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/RunStatus.scala

```diff
@@ -18,13 +18,16 @@ object RunStatus {
 
   sealed trait TerminalRunStatus extends RunStatus {
     def eventList: Seq[EventStartTime]
+    def machineType: String
+    def zone: String
+    def instanceName: String
   }
 
-  case class Success(eventList: Seq[EventStartTime]) extends TerminalRunStatus {
+  case class Success(eventList: Seq[EventStartTime], machineType: String, zone: String, instanceName: String) extends TerminalRunStatus {
     override def toString = "Success"
   }
 
-  final case class Failed(errorCode: Int, errorMessage: Option[String], eventList: Seq[EventStartTime])
+  final case class Failed(errorCode: Int, errorMessage: Option[String], eventList: Seq[EventStartTime], machineType: String, zone: String, instanceName: String)
```

Comment:
Unless I missed something, nothing ever reads this triple for Failed so they could just not be here.

OTOH, There might be value in sending them to metadata even in case of a failure, so you can still calculate the cost for failed jobs